.travis.yml

```diff
@@ -34,7 +34,7 @@ env:
   global:
     - secure: "MOw+6pAQ8s7XX1U9Pbgw8SIuz5MHODj6PXe6xCSIyGR14/0KFNiR9s6NEIa9i7qnp7HOq/V2M9ZFPPioQCrL8k5V4XzZfNz6kXGUT8DoSAAnAAnDWy4yMRxYMlEcjSY8l0KZ9B3szxypkLh0Qd9+pHZYvRSLt8A8UcZSVSNGDjg="
     - WHEELHOUSE="https://unidata-python.s3.amazonaws.com/wheelhouse/index.html"
-    - WHEELDIR="wheelhouse/"
+    - WHEELDIR="$HOME/wheelhouse/"
```

Comment:
Looks like this filepath isn't quite right

Reply:
Actually, needed to remove the $PWD elsewhere since now WHEELDIR is a full path.